front/app/components/admin/ActionForm/FlowVisualization/Blocks/VerificationModal.tsx

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+
+import {
+  Box,
+  Title,
+  Text,
+  Icon,
+  fontSizes,
+  colors,
+} from '@citizenlab/cl2-component-library';
+
+import useVerificationMethod from 'api/verification_methods/useVerificationMethod';
+
+import useLocalize from 'hooks/useLocalize';
+
+import Modal from 'components/UI/Modal';
+
+import { FormattedMessage } from 'utils/cl-intl';
+
+import messages from './messages';
+import { getVerifiedDataList } from './utils';
+
+interface Props {
+  opened: boolean;
+  onClose: () => void;
+}
+
+const VerificationModal = ({ opened, onClose }: Props) => {
+  const { data: verificationMethod } = useVerificationMethod();
+  const localize = useLocalize();
+
+  const verificationMethodMetadata =
+    verificationMethod?.data.attributes.action_metadata;
+
+  if (!verificationMethodMetadata) return null;
+
+  const verifiedDataList = getVerifiedDataList(
+    verificationMethodMetadata,
+    localize
+  );
+
+  return (
+    <Modal
+      opened={opened}
+      close={onClose}
+      niceHeader={true}
+      header={
+        <Title ml="20px" variant="h3" color="primary">
+          <FormattedMessage
+            {...messages.xVerification}
+            values={{
+              verificationMethod: verificationMethodMetadata.name,
+            }}
+          />
+        </Title>
+      }
+      closeOnClickOutside={false}
+      width={'550px'}
+    >
+      <Box m="20px">
+        <Box>
+          <Text fontWeight="bold">
+            <FormattedMessage {...messages.dataReturned} />
+          </Text>
+          <ul>
+            {verifiedDataList.map((attribute, index) => (
+              <li
+                key={index}
+                style={{ fontSize: fontSizes.s, marginBottom: '8px' }}
```

Comment:
Let's avoid inline styling like this when we can? While I don't think it's the worst thing in the world, it circumvents styled components therefore no cross browser support. Also, I'm pretty sure adding styles like this prevents them from being minified which is not great. It can also be very easily avoided.